repro/ResponseContext.cxx

```diff
@@ -1110,7 +1111,8 @@ ResponseContext::processResponse(SipMessage& response)
          if(mRequestContext.getOriginalRequest().method() == INVITE)
          {
             auto currentTargetPtr = i->second;
-            mRequestContext.updateTimerC(currentTargetPtr->mSipMessageOptions.mTxOptions.mTimerC);
+            int delay = currentTargetPtr->mSipMessageOptions ? currentTargetPtr->mSipMessageOptions->mTxOptions.mTimerC : 0;
```

Comment:
rename to delay to customDelayMs so that a value of 0 makes more sense when looking at this code :)